go.mod

```diff
@@ -6,14 +6,14 @@ require (
 	github.com/diktyo-io/appgroup-api v0.0.9-alpha
 	github.com/diktyo-io/networktopology-api v0.0.8-alpha
 	github.com/dustin/go-humanize v1.0.0
-	github.com/google/go-cmp v0.5.5
+	github.com/google/go-cmp v0.5.7
```

Comment:
perhaps rephrase the commit message as "upgrade go-cmp and gonum packages"?

Reply:
:+1: